src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,368 @@
+| RawSources   | Same as defined in [Introduction][intro], but elevated from OPTIONAL to REQUIRED                                                               |
+| Type         | RECOMMENDED. Short identifier of the mask. Reserved values: `Brain` - brain mask, `Lesion` - lesion mask, `Face` - face mask, `ROI` - ROI mask |
+| Resolution   | REQUIRED if `res` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the resolution keyword.         |
+| Density      | REQUIRED if `den` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the density keyword.            |
+
+Examples:
+
+```Text
+func_loc/
+    sub-001/
+        func/
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.nii.gz
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.json
+```
+
+```Text
+manual_masks/
+    sub-001/
+        anat/
+            sub-001_desc-tumor_mask.nii.gz
+            sub-001_desc-tumor_mask.json
+```
+
+## Segmentations and parcellations
+
+Common JSON metadata fields:
+
+| **Key name** | **Description**                                                                                                                        |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| Manual       | OPTIONAL. Boolean. Indicates if the segmenation was performed manually or via an automated process                                     |
+| Atlas        | OPTIONAL. Which atlas (if any) was used to derive the segmentation.                                                                    |
+| Resolution   | REQUIRED if `res` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the resolution keyword. |
+| Density      | REQUIRED if `den` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the density keyword.    |
+
+### Discrete Segmentations
+
+Discrete segmentations of brain tissue represent each tissue class with a unique
+integer label in a 3D volume. See [Anatomical Labels](#anatomical-labels) for interpretation how
+integer values map to tissue classes.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat|func|dwi/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>]_dseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_dseg.nii.gz
+            sub-001_space-orig_dseg.json
+```
+
+A segmentation could be a binary mask that functions as a discrete `label` for a
+single structure. In this case, the label key must be used to specify the
+corresponding structure. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-GM_dseg.nii.gz
+```
```

Comment:
Given one can have a mask "`_mask.nii[.gz]`" image with "`"Type" = "ROI"`", this seems to present two different ways of representing the same underlying information. Personally I would be utilising "`_mask`" for anything binary, and only using "`_dseg`" if there is *more than one* unique integer value present.

Reply:
This would be my default posture, as well, but I recall at the derivatives workshop that the relationship between atlases, masks and segmentations were discussed extensively, so I have to assume that there is a good reason for this. @ahoopes @edickie @chrisgorgo or @tyarkoni might remember the conversations that led to this decision, so I will ping them.

If I can take a guess, it would be that the lookup table that defines labels is specified for `dseg`, but extending it to `mask` ended up being more complicated than it was worth.

Reply:
I think https://github.com/bids-standard/bids-specification/pull/265#discussion_r432874774 was meant to go here, so linking to keep the conversation intact.

I will make a pull request for `dseg` being integer, `mask` being binary, and `probseg` being continuous from 0 to 1.